app/notifications/process_notifications.py

```diff
@@ -148,18 +149,30 @@ def persist_notification(
         updated_at=updated_at,
     )
     if notification_type == SMS_TYPE:
-        if service.has_permission(SMS_TO_UK_LANDLINES):
+        try:
             phonenumber = PhoneNumber(recipient)
-            phonenumber.validate(allow_international_number=True, allow_uk_landline=True)
+            phonenumber.validate(
+                allow_international_number=True, allow_uk_landline=service.has_permission(SMS_TO_UK_LANDLINES)
+            )
             formatted_recipient = phonenumber.get_normalised_format()
             recipient_info = phonenumber.get_international_phone_info()
-        else:
-            formatted_recipient = validate_and_format_phone_number(recipient, international=True)
-            recipient_info = get_international_phone_info(formatted_recipient)
-        notification.normalised_to = formatted_recipient
-        notification.international = recipient_info.international
-        notification.phone_prefix = recipient_info.country_prefix
-        notification.rate_multiplier = recipient_info.rate_multiplier
+            notification.normalised_to = formatted_recipient
+            notification.international = recipient_info.international
+            notification.phone_prefix = recipient_info.country_prefix
+            notification.rate_multiplier = recipient_info.rate_multiplier
+        except InvalidPhoneError as e:
+            if job_id:
+                formatted_recipient = recipient
+                notification.normalised_to = formatted_recipient
+                notification.international = False
+                notification.phone_prefix = "+44"
+                notification.rate_multiplier = 0
+                notification.billable_units = 0
+                notification.status = NOTIFICATION_VALIDATION_FAILED
+                notification.updated_at = datetime.now(UTC)
```

Comment:
I know the way we normally do UTC datetimes (`datetime.utcnow()`) is deprecated, but using `datetime.now(UTC)` instead changes the behaviour since the datetime object is no longer naive. It's storing dates in the database as BST in summer, so the test that covers this line fails when the date is in July.

Perhaps we can we use the deprecated form for now, and then make getting rid of the deprecated format everywhere a separate ticket on a backlog so that it doesn't block this work.